Project.toml

```diff
@@ -34,4 +35,5 @@ SparseArrays = "1"
 StaticArrays = "1"
 SummationByPartsOperators = "0.5.41"
 TimerOutputs = "0.5.7"
+TrixiBase = "0.1"
```

Comment:
```suggestion
TrixiBase = "0.1.1"
```
We require the version including your fix for doctests

Reply:
Yes, thanks!